.buildkite/pipelines/code_coverage/daily.yml

```diff
@@ -0,0 +1,41 @@
+env:
```

Comment:
I would recommend moving these Slack notification settings to the pipeline yaml at buildkite.com, e.g. here https://buildkite.com/elastic/kibana-code-coverage-main/steps

If, for example, this yml file can't be parsed when calling `buildkite-agent pipeline upload` you won't be notified about it. You will, however, if it's in the other yml.

Reply:
Ahh, thanks so much!